src/lang/english.txt

```diff
@@ -3473,10 +3477,13 @@ STR_GROUP_DEFAULT_AIRCRAFTS                                     :Ungrouped aircr
 STR_GROUP_COUNT_WITH_SUBGROUP                                   :{TINY_FONT}{COMMA} (+{COMMA})
 
 STR_GROUPS_CLICK_ON_GROUP_FOR_TOOLTIP                           :{BLACK}Groups - click on a group to list all vehicles of this group. Drag and drop groups to arrange hierarchy.
-STR_GROUP_CREATE_TOOLTIP                                        :{BLACK}Click to create a group
+STR_GROUP_CREATE_TOOLTIP                                        :{BLACK}Click to create a group. Drag and drop vehicle to create a group for it with automatic name. Ctrl+Drag to add shared vehicles.
```

Comment:
```suggestion
STR_GROUP_CREATE_TOOLTIP                                        :{BLACK}Click to create a group. Drag and drop a vehicle to create a group for it with an automatic name. Ctrl+Drag to add shared vehicles.
```
what's a shared vehicle?

Reply:
Vehicles with shared orders. I hope it's better now.